### ISSUE TITLE
Offline Maps (Caching)

### ISSUE THREAD

**Issue by Jacki3** [LoGaCulture/LUTE#25]
**Is your feature request related to a problem? Please describe.**
Many sites, including Avebury, lack good connectivity and applications that rely on this could be hindered. In our case, the map features pull data remotely then cache this. The issue will be that the map may not load (or will take time to load) potentially spoiling the experience for players.

**Describe the solution you'd like**
Maps should be downloaded per area and cached on first download. The app should come with cached map tiles (designers should specify which areas are required).

**Additional context**
This is already possible using Mapbox but we need finer controls and to increase the ease of use.
